xml2kinto/__main__.py

```diff
@@ -172,7 +194,9 @@ def main(args=None):
             xpath='certItems/*',
             kinto_client=kinto_client,
             bucket=args.cert_bucket,
-            collection=args.cert_collection),
+            collection=args.cert_collection,
+            schema=schemas.get(args.cert_bucket,
+                               schemas.get(CERT_COLLECTION))),
```

Comment:
What's the idea behind looking for the bucket name first, and then the collection name?